.github/workflows/wheels.yml

```diff
@@ -175,9 +177,11 @@ jobs:
           fi
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@faf86a6ed7efa889faf6996aa23820831055001a  # v2.23.3
+        uses: pypa/cibuildwheel@90a0ddeff0f23eebc21630e65d66d0f4955e9b94  # v3.0.0b1
         env:
           CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat[1] }}
+          # delete when we switch 3.14(t)-dev to 3.14(t)
+          CIBW_ENABLE: cpython-prerelease
```

Comment:
This is unnecessary, as is the change in `pyproject.toml`. It was already set up correctly.

Reply:
Probably my fault, I thought this was being added in this PR when I gave a suggestion, didn’t realize it was already there.

Reply:
No worries. Our setup here is a bit unusual, we're explicitly iterating over all platforms and Python versions, so `cpython-prerelease` doesn't do anything (and hence can always be left in) unless `"cp31x"` is added with `x` for an unreleased version.